scalastyle-config.xml

```diff
@@ -258,7 +258,7 @@ This file is divided into 3 sections:
     scala.collection.JavaConverters._ and use .asScala / .asJava methods</customMessage>
   </check>
 
-  <check customId="commonslang2" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
+  <check customId="commonslang2" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
```

Comment:
@srowen and @databricks-david-lewis . This was the root cause.
I'm not sure if there is any change inside `TokenChecker`, but `RegexChecker` is better for `org.apache.commons.lang.` since it's not a token.